File: julia/quick_sort.jl
quick_sort  = function (array::Array)
    if length(array) <= 1
        return array
    end
    pivot = array[begin]
    less = quick_sort([i for i in array[2:length(array)] if i <= pivot])
    greater = quick_sort([i for i in array if i > pivot])

    return vcat(less, pivot, greater)
end

File: python/quick_sort.py
#%%
def quick_sort(array):
    if len(array) <= 1:
        return array
    pivot = array[0]
    less = quick_sort([i for i in array[1:] if i <= pivot])
    greater = quick_sort([i for i in array if i > pivot])
    return less + [pivot] + greater
#%%

